cypher-shell/src/main/java/org/neo4j/shell/cli/InteractiveShellRunner.java

```diff
@@ -35,7 +35,7 @@
 public class InteractiveShellRunner implements ShellRunner, SignalHandler {
     static final String INTERRUPT_SIGNAL = "INT";
     private final static String FRESH_PROMPT = "> ";
-    private final static AnsiFormattedText CONTINUATION_PROMPT = AnsiFormattedText.s().bold().append("  ");
+    private final static AnsiFormattedText CONTINUATION_PROMPT = AnsiFormattedText.s().bold().append("       ");
```

Comment:
Now that we do not have the newline this prompt will not align the lines anymore, since the prompt has a variable length. We should either have no spaces at all here or do something more clever (not static, based on the length of the actual prompt - but the problem is if the prompt is too long compared to the width of the terminal, in which case we should either skip indentation here or add a newline to the prompt).